src/mrpro/phantoms/_EllipsePhantom.py

```diff
@@ -91,7 +91,7 @@ def image_space(self, image_dimensions: SpatialDimension[int]) -> torch.Tensor:
         """
         # Calculate image representation of phantom
         ny, nx = image_dimensions.y, image_dimensions.x
-        ix, iy = torch.meshgrid(
+        ix, it = torch.meshgrid(
```

Comment:
why `it`? also in line 104 below

Reply:
spelling autocorrect fail of the precommit .... iy->it
we should add this as an expectation....

Reply:
should be fixed now